### PR TITLE
Add ability to set personalizations

### PR DESCRIPTION
**Pull request by MaxLustig**

### CODE REVIEW

File: lib/sendgrid_actionmailer.rb [hunk 1]
```diff
@@ -75,6 +81,22 @@ def to_emails(input)
       end
     end
 
+    def to_personalization(personalization)
+      Personalization.new.tap do |p|
```
Comment (walski) [inline lib/sendgrid_actionmailer.rb]: This really should be `e` if it's a one char variable (as much as you hate that ;)) or `personalization`

Comment (walski) [reply lib/sendgrid_actionmailer.rb]: `p` is just something different in Ruby :)

Comment (MaxLustig) [reply lib/sendgrid_actionmailer.rb]: Unfortunately we already have `personalization`, although I could switch them up. `p` is used further down the file so I wanted to keep it consistent.

Comment (walski) [reply lib/sendgrid_actionmailer.rb]: ok

**Pull request merged**